client/src/domain/Seat/style.scss

```diff
@@ -0,0 +1,10 @@
+.hall__seat {
+  width: 20px;
+  height: 20px;
+  background: grey;
+  outline: none;
+  border: none;
+}
+.selected {
```

Comment:
Should probably create a class or modifier for the unavailable seat, if we stick to scss, but since we are planning styled components, I am not sure.